samples/compiler_plugins/xnnpack/test/fully-connected-e2e.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: iree-compile --iree-hal-target-backends=llvm-cpu --iree-plugin=xnnpack %s | \
+// RUN: iree-run-module \
+// RUN:     --device=local-sync \
+// RUN:     --executable_plugin=$IREE_BINARY_DIR/samples/custom_dispatch/xnnpack/plugin/system_plugin$IREE_DYLIB_EXT \
+// RUN:     --module=- \
+// RUN:     --function=main \
+// RUN:     --input=1x2x8xi8=1 \
+// RUN:     --input=4x8xi8=2 \
+// RUN:     --input=4x8xi8=8 | \
+// llvm-lit does not like checking for '[['
+// RUN:     sed 's/\[/(/g' | tee /tmp/dump.txt | \
```

Comment:
Remove the tee command since it can cause a race condition if the same file is used somewhere else.

Reply:
Whoops, this was for debugging. Not meant to make it into the final PR